ocaml/squeezed/lib/squeeze.ml

```diff
@@ -210,7 +203,8 @@ module Stuckness_monitor = struct
 
   (** Update our internal state given a snapshot of the outside world *)
   let update (x : t) (host : host) (now : float) =
-    List.iter
+    let domains = host.domains in
```

Comment:
Any benefits with this extra variable?

Reply:
This is bad habit of mine: although that has any obvious benefits, if a value occurs two or more times, I want to bind a variable to it.

I am improving on this habit.

Reply:
It's fine. Being a field means it's already available in a pre-calculated form, so there should be no need for an extra binding